src/framework/dropdown/DropdownGroup.spec.jsx

```diff
@@ -0,0 +1,21 @@
+
+import { TestCaseFactory } from 'react-test-kit';
+
+import Dropdown from './Dropdown.jsx';
+import DropdownGroup from './DropdownGroup.jsx';
+
+describe('DropdownGroup', () => {
+  const requiredProps = {
+    options: [['blue', 'pink'], ['black', 'white']],
+    optionsClass: ['', 'dropdownClass'],
+    labelProvider: () => undefined,
+  };
+
+  describe('DOM structure', () => {
+    it('is a Dropdown', () => {
+      const props = Object.assign({}, requiredProps);
+      const testCase = TestCaseFactory.create(DropdownGroup, props);
+      expect(testCase.findComponents(Dropdown)).toBeDefined();
```

Comment:
`findComponents` return a list. `[]` will also be defined. So not rendering a `Dropdown` is not caught by the test.